web/src/locales/ar/data.json

```diff
@@ -595,7 +595,9 @@
     "Edit Model": "Edit Model",
     "Model text": "Model text",
     "Model text - Tooltip": "Casbin access control model, including built-in models like ACL, RBAC, ABAC, RESTful, etc. You can also create custom models. For more information, please visit the Casbin website",
-    "New Model": "New Model"
+    "New Model": "New Model",
+    "Basic Editor": "Basic Editor",
```

Comment:
The order is wrong. Update it with: https://casdoor.org/docs/contributing/#how-to-update-i18n-data